CRUD/delete.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+include "../configuration/db.php"; 
+
+$productId = $_GET['id'];
+
+$delete_query = "DELETE FROM Product WHERE `Product`. product_id = '$productId'";
```

Comment:
db tables name, and column name should be in lower case and can contain _ to combine words.